service/controller/v13/encrypter/vault/vault.go

```diff
@@ -291,9 +291,16 @@ func (e *Encrypter) AddAWSIAMRoleToAuth(vaultRoleName, iamRoleARN string) error
 	}
 
 	if role.BoundIAMRoleARN == "" {
-		role.BoundIAMRoleARN = iamRoleARN
-	} else if !strings.Contains(role.BoundIAMRoleARN, iamRoleARN) {
-		role.BoundIAMRoleARN = fmt.Sprintf("%s,%s", role.BoundIAMRoleARN, iamRoleARN)
+		e.logger.Log(fmt.Sprintf("empty bound_iam_role, setting %q", strings.Join(iamRoleARNs, ",")))
```

Comment:
I don't follow the log structure here. We should just use the formats we have established. The level and message pairs are missing.

Reply:
This is just for debugging, will be gone soon.

Reply:
Already removed.